src/main/scala/BootAddrReg.scala

```diff
@@ -15,7 +15,7 @@ import freechips.rocketchip.prci._
 
 case class BootAddrRegParams(
   defaultBootAddress: BigInt = 0x80000000L, // This should be DRAM_BASE
-  bootRegAddress: BigInt = 0x4000,
+  bootRegAddress: BigInt = 0x1000,
```

Comment:
If you make this change, you have to change here as well: https://github.com/ucb-bar/testchipip/blob/307fca3c3ec4484d071744df6752bbbb77be7ddb/src/main/resources/testchipip/csrc/cospike_impl.cc#L198